packages/wdio-testingbot-service/testingbot-service.d.ts

```diff
@@ -9,6 +9,48 @@ declare module WebdriverIO {
     }
 }
 
+interface TunnelLauncherOptions {
+    // The TestingBot API key which you can get for free, listed in our member area
+    apiKey: string;
+ 
+    // The TestingBot API secret which you can get for free, listed in our member area
+    apiSecret: string;
```

Comment:
Question: Are these options now mandatory? Is there no way to grab the ones from `user`/`key`? (Also it seems to work when not passing any options.)